src/viking.js

```diff
@@ -1,15 +1,122 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health=health;
+    this.strength=strength;
+    
+  } // -- constructor
+attack (){
+  return this.strength
+}
+receiveDamage(damage){
+  this.health = this.health - damage
+  return 
```

Comment:
también una forma de asignar una operación es usando esta sintaxis this.health -= damage (-= o += dependiendo)